packages/IMAPClient-Core.package/ICEndPoint.class/instance/logout.st

```diff
@@ -1,4 +1,8 @@
 login
 logout
 
-	^ self connectionHandler logout
```

Comment:
If this is in ICEndPoint now, we should probably remove the corresponding method from ICConnectionHandler.